### DISCUSSION TITLE
Your organization does not have enough balance to request this action ...

### DISCUSSION THREAD

**Discussion by bijlevel** [dmarx/video-killed-the-radio-star#63] [Q&A] [answered]
When running to notebook I get the "error" message "Your organization does not have enough balance to request this action." Does it mean I (just guessing) need to buy credits for Dreamstudio in order to have a positive balance? Or am I missing something? (I do have an API key...)

**Comment by dmarx** [comment, marked as answer]
It means you are out of dreamstudio credits, yes. you can either purchase credits, or you can uncheck the box at the top of the notebook to download the checkpoint from huggingface and run inference 'locally' in the notebook (i.e. using google colab's compute).

**Reply by bijlevel** [reply]
Thanks!
